### PR TITLE
OCM-9817 | test: automate id:75210 delete byo oidc config cluster via rosacli

### PR DESCRIPTION
**Pull request by yuwang-RH**
Automate [OCM-9817](https://issues.redhat.com//browse/OCM-9817) | test: automate id:75210 delete byo oidc config via rosacli
Add OIDC Endpoint URL in the cluster_detail json file to be used by case
log : https://privatebin.corp.redhat.com/?0f1091b94db14286#GGEnV2oZ7wqiTsapzXYr8YFKi7MDrZh7yKAye6SwpFdP

### CODE REVIEW

File: tests/e2e/test_rosacli_cluster_post.go [hunk 1]
```diff
@@ -349,3 +352,89 @@ var _ = Describe("Create cluster with the version in some channel group testing"
...
+			clusterService     rosacli.ClusterService
+			clusterID          string
+			err                error
+			awsClient          *aws_client.AWSClient
+			oidcEndpointUrlC   string
+			clusterConfig      *config.ClusterConfig
+			oidcConfigC        string
+			oidcProviderArn    string
+			profile            *profilehandler.Profile
+		)
```
[the remark this answers is not in the document]
Comment (yuwang-RH) [reply tests/e2e/test_rosacli_cluster_post.go:371]: "All common variables are declared in high level and can be used by others cases in the same blocker.
profile is used for the condition of oidc config which is a matrix defined in profile. And clusterConfig is a struct parsing the clusterConfig file in shared_dir, both of them are needed for the case." I didn't get what concern you have, could you explain more about the risk if you have any?

**Comment by yuwang-RH** [comment]
@yasun1 Could you help review again? I reply your comment and did the rebase to solve the conflict

File: tests/e2e/test_rosacli_cluster_post.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (yasun1) [reply tests/e2e/test_rosacli_cluster_post.go:395]: I think I have no concerns now for we have finished the same discussion in https://github.com/openshift/rosa/pull/2272

**Pull request merged**